ARE/middleware/src/main/java/eu/asterics/mw/services/AstericsModelExecutionThreadPool.java

```diff
@@ -122,25 +117,26 @@ public Thread newThread(Runnable r) {
     /*
      * = Executors .newSingleThreadExecutor(new ThreadFactory() {
      * 
-     * @Override public Thread newThread(Runnable arg0) { logger.fine(
-     * "Creating Thread: "+MODEL_EXECUTOR_LIFECYCLE);
+     * @Override public Thread newThread(Runnable arg0) { logger.fine( "Creating Thread: "+MODEL_EXECUTOR_LIFECYCLE);
      * 
-     * Thread newThread=Executors.defaultThreadFactory().newThread(arg0);
-     * newThread.setName(MODEL_EXECUTOR_LIFECYCLE); return newThread; } });
+     * Thread newThread=Executors.defaultThreadFactory().newThread(arg0); newThread.setName(MODEL_EXECUTOR_LIFECYCLE); return newThread; } });
      */
     private AstericsModelExecutionThreadPool() {
-        TASK_SUBMIT_TIMEOUT = new Integer(
-                AREProperties.instance.getProperty(TASK_SUBMIT_TIMEOUT_PROPERTY, String.valueOf(TASK_SUBMIT_TIMEOUT)));
+        AREProperties.instance.setDefaultPropertyValue(TASK_SUBMIT_TIMEOUT_PROPERTY, String.valueOf(DEFAULT_TASK_SUBMIT_TIMEOUT),
+                "This timeout in [ms] is used to start/stop a model and do other tasks related to models. If you get a timeout during ARE/model start try to increase the value. Especially on a Raspberry Pi this can be necessary.");
+        try {
+            TASK_SUBMIT_TIMEOUT = new Integer(AREProperties.instance.getProperty(TASK_SUBMIT_TIMEOUT_PROPERTY));
+        } catch (NumberFormatException e) {
+            TASK_SUBMIT_TIMEOUT = DEFAULT_TASK_SUBMIT_TIMEOUT;
```

Comment:
this is not necessary since `TASK_SUBMIT_TIMEOUT` is already set to default value at initialization.

Reply:
hmm, yes your are right. On the other just to be sure, if there is a parsing error, set it to the default value :-)

Reply:
yeah, nevermind. it's not needed, but an empty catch-clause maybe also would look akward ;)